scripts/setup_aimsun.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+# If the following error is encountered upon installation:
+# 'PackagesNotFoundError: The following packages are not available from current channels: - python=2.7.4'
+# Run the two lines of code below:
+
+# conda config --append channels https://repo.anaconda.com/pkgs/free
+# conda config --append channels https://repo.anaconda.com/pkgs/pro
```

Comment:
I would just go ahead and uncomment these. Everyone who tries using this will experience the issue